### ISSUE TITLE
Resolution

### ISSUE THREAD

**Issue by akifai** [EngoEngine/TrafficManager#27]
The resolution of the image is greater than the resolution of the created area. So, at start, the picture does not fit into the window.

**Comment by Noofbiz** [comment]
Yeah, I'm not sure when that changed, but the hello-world one is set to 400x400 and it doesn't update to 800x800 in part 4 to accomidate the HUD. The tutorial at that point is just to blit an image to the screen, so I don't see that as an issue, per-se, but the tutorial does mention the background color changing, which you can't see at the current resolution. I guess just play around with it if you want, I don't think it's much of a thing and the next tutorial handles it by using the RenderComponent.Scale property.
